tests/integration_tests/cmd/test_clean.py

```diff
@@ -39,9 +39,7 @@ class TestCleanCommand:
     def test_clean_rotated_logs(self, class_client: IntegrationInstance):
         """Clean with log params alters expected files without error"""
         assert class_client.execute("cloud-init status --wait").ok
-        assert class_client.execute(
-            "logrotate /etc/logrotate.d/cloud-init.logrotate"
-        ).ok
+        assert class_client.execute("logrotate /etc/logrotate.d/cloud-init").ok
```

Comment:
Upps. I am sorry about this. Didn't recognized that i missed removing the filename extension as `dh_logrotate` just use it for detecting logrotate files during package installation.